app/presenters/confirmation_email_addresses_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ConfirmationEmailAddressesPresenter < Struct.new(:claim)
+  delegate :primary_claimant, :representative, :secondary_claimants, to: :claim
+
+  def self.for(claim)
```

Comment:
I'd be inclined to rename this to something like `email_addresses_for`, as it's not otherwise clear what it returns.